euth/users/models.py

```diff
@@ -197,6 +197,7 @@ def signup(self, username, email, timezone, commit=True):
         """Update the fields required for sign-up."""
         self.username = username
         self.email = email
-        self.timezone = timezone
+        if timezone in common_timezones:
```

Comment:
To me this commit seems superfluous. AFAIK django validates the form input against the model. @slomo do you have an opion about this?